.projenrc.ts

```diff
@@ -9,6 +10,7 @@ const project = new CdklabsTypeScriptProject({
   name: 'cdk-cloudformation',
   projenrcTs: true,
   sampleCode: false,
+  npmAccess: NpmAccess.PUBLIC,
```

Comment:
This is the only manual change I made. Others are because of upstream projen settings changes.